### PR TITLE
Minified Cesium correctly

### LINKED ISSUE #2915: Cesium is not compressed in ol3cesium file

### PR DESCRIPTION
**Pull request by oterral**
Fix #2915

[Test](https://mf-geoadmin3.dev.bgdi.ch/teo_fix_build/?lang=fr&topic=ech&bgLayer=ch.swisstopo.pixelkarte-farbe&layers=ch.swisstopo.zeitreihen,ch.bfs.gebaeude_wohnungs_register,ch.bafu.wrz-wildruhezonen_portal,ch.swisstopo.swisstlm3d-wanderwege&layers_visibility=false,false,false,false&layers_timestamp=18641231,,,&dev3d=true&lon=8.22546&lat=42.60866&elevation=541694&heading=360.000&pitch=-44.250)

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -200,8 +200,7 @@ ol3cesium: ol .build-artefacts/ol3-cesium
 	git show; \
 	ln -T -f -s ../../../../ol3-cesium-plugin/ src/plugins/geoadmin; \
 	( cd cesium; [ -f node_modules/.bin/gulp ] || npm install ); \
-	( cd cesium; if [ -f "Build/Cesium/Cesium.js" ] ; then echo 'Skipping Cesium minified build'; else node_modules/.bin/gulp minifyRelease; fi ); \
-	( cd cesium; if [ -f "Build/CesiumUnminified/Cesium.js" ] ; then echo 'Skipping Cesium debug build'; else node_modules/.bin/gulp combine; fi ); \
+	( cd cesium; if [ -e "Build/Cesium" ] && [ -e "Build/CesiumUnminified" ]; then echo 'Skipping Cesium debug build'; else npm run combine; rm -rf Build/Cesium; npm run minifyRelease; fi ); \
```
Comment (gberaudo) [inline Makefile]: @oterral, what was the problem in the previous code?

**Comment by gberaudo** [comment]
The test link returns a 404.

File: Makefile (hunk 1 above)
Comment (oterral) [inline Makefile]: The `combine` target replaced the  minified version of cesium in 'Build/Cesium/Cesium.js' by an unminified one

**Comment by oterral** [comment]
The branch was deploying is good now.

File: Makefile (hunk 1 above)
Comment (gberaudo) [inline Makefile]: I see.
The drawback of your PR is that we now always compile Cesium twice.
When developping, I often removed manually the combined flavour of Cesium to only pay one compilation.
There may be a trick, by prepending another target first, like this (not tested):
`node_modules/.bin/gulp generateDocumentation combine`.

We can try to optimize later, so +1 for merging.

Comment (oterral) [inline Makefile]: Yes we compile twice, not ideal. We'll see in future.

**Pull request merged**